tests/integration/test.replication.js

```diff
@@ -729,6 +729,136 @@ adapters.forEach(function (adapters) {
       }).catch(done);
     });
 
+    it('Test replicate resumes OK', function (done) {
+      var db = new PouchDB(dbs.name);
+      var remote = new PouchDB(dbs.remote);
+
+      db.bulkDocs({ docs: docs.slice(0, 1) })
+        .then(function () {
+          PouchDB.replicate(db, remote)
+            .on('error', done)
+            .on('complete', function (result) {
+              result.docs_read.should.equal(1);
+              result.docs_written.should.equal(1);
+              result.initial_checkpoint.should.equal(0);
```

Comment:
as above, suggest capturing the `since` parameter sent to `changes()` rather than changing the schema of `result`